redis/RedisCluster.php

```diff
@@ -209,10 +209,10 @@ public function msetnx(array $array) { }
     /**
      * Remove specified keys.
      *
-     * @param   int | array $key1 An array of keys, or an undefined number of parameters, each a key: key1 key2 key3
-     *                            ... keyN
-     * @param   string      $key2 ...
-     * @param   string      $key3 ...
+     * @param   int|array|string    $key1 An array of keys, or an undefined number of parameters, each a key: key1 key2 key3
+     *                              ... keyN
+     * @param   string              $key2 ...
```

Comment:
It's better to avoid unnecessary formatting changes

Reply:
👌